src/components/sound-picker/sound-picker.tsx

```diff
@@ -41,6 +155,7 @@ export const SoundPicker = (props: ISoundPickerProps) => {
     <div className="sound-picker-container">
       <div className="sound-picker-select-container">
         <select className="sound-picker"
+          ref={soundSelectRef}
```

Comment:
This won't be necessary, instead you'll probably want something like:
```
disabled={sunndSelectDisabled}
```

Reply:
Also done (see above comment response).